### ISSUE TITLE
Attaching avatars to things other than Users

### ISSUE THREAD

**Issue by kristofer** [jazzband/django-avatar#89]
Is there a way to attach avatars to other things than Users? I'd like to use it to attach avatars (well, company logos) to my Company objects (and I really like the mgmt stuff you bring.)

**Comment by lggwettmann** [comment]
Same question here. That would be super useful if we could use django-avatar for our user avatars and other models as well.

**Comment by anuj9196** [comment]
Same question here. I too need this feature. In my case, I have a model to store Person information and want to get gravatar from the email of the person or allow to upload one just like we do for User model.

**Comment by johanneswilm** [comment]
Yes, if any of you @anuj9196 @lggwettmann @kristofer are willing to write a PR for this, I will review.
